index.html

```diff
@@ -3,23 +3,33 @@
   <head>
     <meta charset="utf-8" />
     <title>Rock Paper Scissors</title>
+    <link rel="stylesheet" href="./style.css" />
   </head>
 
   <body>
+    <h1>Rock Paper Scissors Game</h1>
+    <div class="flex-container-1">
+      <button class="btn flex-item rock">Rock</button>
+    </div>
+    <div class="flex-container-2">
+      <button class="btn flex-item paper">Paper</button>
+      <button class="btn flex-item scissors">Scissors</button>
+    </div>
+    <div>
+      <p></p>
+    </div>
+
     <script>
-      /* create variables to store cpu and player wins */
       let cpuWins = 0;
       let playerWins = 0;
-
-      /*create variable to store cpu choice as rock, paper or scissors */
       let cpuChoice;
 
-      /*create variable to choose a random number then store it in a variable */
-      /*change value of cpu choice variable depending on value of random number variable */
-      /*create function to run this code and make a different cpu choice each time function is run */
+      const score = document.querySelector("p")
```

Comment:
what happens when we have multiple `p` elements on the page?

Reply:
it will be a node list and I have to specify which P I want to select?

Reply:
what are some other ways you can select a specific element

Reply:
I could give the element an ID and select it that way, or change its class

Reply:
get elementByID or get elementByClassName or get elementByTagname?

Reply:
when would you use a class vs id?

Reply:
Class if I had multiple similar elements and wanted to style them together and ID if it was a unique identifier I just needed to use once?

Reply:
yee